### PR TITLE
LPS-83168 Generate Pom Info for all portal jars

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
cc @petershin

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java [hunk 1]
```diff
@@ -4166,6 +4255,9 @@ private boolean _syncReleaseVersions(
 
 	private static final char _DEPENDENCY_KEY_SEPARATOR = '/';
 
+	private static final String _GENERATE_POM_INFO_TASK_NAME =
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java]: Task names are public APIs (changing them is a breaking change), so the constant should be public. After making this change, run `gradlew baseline`.

File: modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java [hunk 2]
```diff
@@ -396,6 +398,8 @@ else if (FileUtil.exists(project, ".lfrbuild-tool")) {
 
 		_addTaskCopyLibs(project);
 
+		Task generatePomInfoTask = _addTaskGeneratePomInfo(project);
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java]: Can't we add this after the `_addTaskDeployDependencies` task, to keep it sorted?

**Review by Ithildir — CHANGES_REQUESTED** [review]
cc @petershin

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
ci:reopen

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/61811.

**Pull request closed without merging**